DotNet/Automation.UI/wwwroot/css/site.css

```
@@ -112,6 +112,55 @@ body { margin-bottom: 60px; background: var(--au-bg); color: #222; }
+  top: 50%;
+  left: 50%;
+  width: 2rem;
+  height: 2rem;
+  margin: -1rem 0 0 -1rem;
+  border: 3px solid #ccc;
+  border-top-color: var(--au-dark, #1f2937);
+  border-radius: 50%;
+  animation: au-card-spin .8s linear infinite;
+  z-index: 6;
+}
+@@keyframes au-card-spin {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@@keyframes" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)